### DISCUSSION TITLE
InfluxDB Dashboard Import Error

### DISCUSSION THREAD

**Discussion by jrdwiz** [jertel/vuegraf#120] [Q&A] [answered]
Hello,

I'm getting the following error when attempting to import the dashboard. (Copy and pasting the json)

Failed to import dashboard: Error: resource_type="check" err="failed to apply resource" metadata_name="competent-hopper-f6f003" err_msg="failed to create checks[\"competent-hopper-f6f003\"]: Could not create task from check: loc 0:0-0:0: unknown operator \"<INVALID_OP>\"\n" metadata_name="noshing-borg-b6f003" err_msg="failed to create checks[\"noshing-borg-b6f003\"]: Could not create task from check: loc 0:0-0:0: unknown operator \"<INVALID_OP>\"\n"

InfluxDB v2.7.1

Please let me know what logs would be helpful.

Any help would be great. 

Thanks

**Comment by jertel** [comment, marked as answer]
Per the project docs/readme:

![image](https://user-images.githubusercontent.com/16639611/236356076-22b25afd-51d9-47c6-90d8-7826c5f97d54.png)
